scipy/optimize/_cobyla_py.py

```diff
@@ -188,9 +191,9 @@ def fmin_cobyla(func, x0, cons, args=(), consargs=None, rhobeg=1.0,
 
 @synchronized
 def _minimize_cobyla(fun, x0, args=(), constraints=(),
-                     rhobeg=1.0, tol=1e-4, maxiter=1000,
-                     disp=False, catol=2e-4, callback=None, bounds=None,
-                     **unknown_options):
+                     rhobeg=1.0, tol=1e-6, maxiter=1000,
+                     disp=0, catol=None, ftarget=-np.inf,
```

Comment:
```suggestion
                     disp=0, catol=None, f_target=-np.inf,
```

The discussions in te `COBYQA` PR converged to this parameter name, let's do the same here.

Reply:
I'll make the change once we've reached a conclusion on the above conversation about tolerance.